### ISSUE TITLE
Make units in history fields UDUNITS compliant

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#973]
The units for history fields are not UDUNITS compliant (the standard used for the CF convention). Fixing this would mean changing almost all the units. There are compliance checkers to make sure CF conventions are followed and the units can be handled. One advantage of UDUNITS compliance is that units that follow the convention can be converted and operations can be done on them.

**Comment by wwieder** [comment]
This sounds nice, and would potentially be helpful over the long-term, but how much time are we looking at?  From previous conversations, I thought it was just a handful of fields that @olyson needed changed for ILAMB analyses.  If changing units for all output is a big time sink, are we better off only  going part way on this?

**Comment by ekluzek** [comment]
Yes, I brought the changes to the few fields in #966 to release and will do the same for master for the short term. I just want to make sure we discuss this as a longer term issue. There's hundred's of fields and many files you have to go over, so I imagine it would take a day to sort it all out. In the end we could likely add a UDUNITS compliance checker though. And units would be consistent which would be good.

I've added the next flag so we'll talk about this at the next meeting.

**Comment by billsacks** [comment]
@dlawrenncar points out that this is tied in with a desire to change all of our field names and units to be CMIP-compliant (i.e., CMORized). That might be worth doing in conjunction with this.

**Comment by ekluzek** [comment]
@rgknox and @glemieux this sounds like an issue in CTSM that relates to what you are doing in FATES https://github.com/NGEET/fates/issues/630.

**Comment by rgknox** [comment]
super cool idea @billsacks
